src/ephemeron/lin_tests.ml

```diff
@@ -0,0 +1,61 @@
+open QCheck
+open Util
+
+module Spec =
+  struct
+    module E = Ephemeron.K1.Make(struct type t = int64 let equal = Int64.equal let hash = Int64.to_int end)
+    type t = string E.t
+    
+    type cmd =
+      | Clear
+      | Add of int' * string
+      | Remove of int'
+      | Find of int'
+      | Find_opt of int'
+      | Find_all of int'
+      | Replace of int' * string
+      | Mem of int'
+      | Length
+      | Clean [@@deriving qcheck, show { with_path = false }]
+    and int' = int64 [@gen fun st -> Gen.nat st |> Int64.of_int]
```

Comment:
It may be a good idea to test with a boxed type like `int64`.
Can you add a comment why you chose to do so?